rl4co/models/nn/dec_strategies.py

```diff
@@ -79,7 +77,10 @@ def pre_decoder_hook(self, td: TensorDict, env: RL4COEnvBase):
 
         # Multi-start decoding: first action is chosen by ad-hoc node selection
         if self.num_starts > 1:
-            action = self.select_start_nodes_fn(td, env, self.num_starts)
+            if self.select_start_nodes_fn is not None:
+                action = self.select_start_nodes_fn(td, env, self.num_starts)
+            else:
+                action = env.select_start_nodes(td, num_starts=self.num_starts)
```

Comment:
[Comment only] ok, I am dumb, I commented above on something you did here already :laughing: